test/utils/nextFrame.js

```diff
@@ -15,8 +15,8 @@
  */
 
 /**
- * Returns a promise that resolves once the browser window has loaded and
- * all load callbacks have finished executing.
+ * Returns a promise that resolves once the browser has run the next
+ * animation frame.
  * @return {Promise<void>}
  */
 export function nextFrame(state) {
```

Comment:
Not changed in this PR but do we need `state` here? Doesn't ever seem to be called with an argument.

```suggestion
export function nextFrame() {
```

Also function name could maybe be clearer (`awaitNextFrame`) but again not changed here so fine to leave as is.